openpype/modules/deadline/plugins/publish/submit_publish_job.py

```diff
@@ -235,6 +236,7 @@ def _submit_deadline_post_job(self, instance, job, instances):
             if mongo_url:
                 environment["OPENPYPE_MONGO"] = mongo_url
 
+
```

Comment:
Doesn't the hound mark bark about this? :) There shouldn't be two empty lines in a function, right?
```suggestion

```